executor/executor_test.go

```diff
@@ -3981,7 +4012,7 @@ func (s *testSuite3) TestUnsignedDecimalOverflow(c *C) {
 	for _, t := range tests {
 		res, err := tk.Exec("insert into t values (?)", t.input)
 		if res != nil {
-			defer res.Close()
+			defer c.Assert(res.Close(), IsNil)
```

Comment:
This statement will close the result set earlier.

Reply:
Thanks for reminding, done.